package.json

```
@@ -9,7 +9,8 @@
     "build:production": "npm run remove-dist && npm run build:static && webpack --mode=production",
     "build:test": "npm run build:production -- --include-tests",
     "build:static": "gulp imagemin",
-    "build:storybook": "build-storybook -c .storybook -o dist && npm run build:static",
+    "build:storybook": "build-storybook -c .storybook -o dist",
```

Let's put these here as previously suggested so that both the build-job and the VRT job benefit from it.
```suggestion
    "build:storybook": "build-storybook -c .storybook -o dist --quiet --loglevel warn",
```